apps/bemo-worker/src/worker.ts

```diff
@@ -34,6 +37,10 @@ export default class Worker extends WorkerEntrypoint<Environment> {
 				context: this.ctx,
 			})
 		})
+		.get('/v1/bookmarks/unfurl', async (request) => {
```

Comment:
i dunno if we need version prefixes since we're only guaranteeing the demo server will work with the latest tldraw release

Reply:
makes sense!

Reply:
i kinda like them just cus they're free and give us room to change our minds about things down the line but i'e removed for now